xc/xc7/tests/soc/ibex/ibex.patch

```diff
@@ -0,0 +1,110 @@
+diff --git a/examples/fpga/artya7/rtl/top_artya7.sv b/examples/fpga/artya7/rtl/top_artya7.sv
+index c26ca45..be19828 100644
+--- a/examples/fpga/artya7/rtl/top_artya7.sv
++++ b/examples/fpga/artya7/rtl/top_artya7.sv
+@@ -11,7 +11,7 @@ module top_artya7 (
+   parameter int          MEM_SIZE     = 64 * 1024; // 64 kB
+   parameter logic [31:0] MEM_START    = 32'h00000000;
+   parameter logic [31:0] MEM_MASK     = MEM_SIZE-1;
+-  parameter              SRAMInitFile = "";
++  parameter              SRAMInitFile = "led.vmem";
```

Comment:
Why is this change required?

Reply:
Currently, It is required since we don't have any option to change the parameter values.
This can be done using `chparam -set SRAMInitFile="led.vmem"` in Yosys after reading the design.